src/Traveler.js

```diff
@@ -1,8 +1,11 @@
 class Traveler {
-  constructor(id, name, travelerType) {
-    this.id = id;
-    this.name = name;
-    this.travelerType = travelerType;
+  constructor(traveler) {
+    this.id = traveler.id;
+    this.name = traveler.name;
+    this.travelerType = traveler.travelerType;
+    this.pastTrips = [];
+    this.currentTrips = [];
+    this.futureTrips = [];
   }
```

Comment:
I may just need to look more; where are you storing the user's pending trips?

Reply:
I was planning on having pending trips show up in "future" and display the status for any upcoming trips. Perhaps adjust style to show pending in red and approved in green.  Since it's a property of the trip I don't see the need for an array in the Traveler class.